packages/config/config/devices/0x0060/ad142.json

```diff
@@ -1,18 +0,0 @@
-// Everspring AD142
```

Comment:
What happened here? Merged with another one?

Reply:
Not sure but I re-added it. Had to have been my manual mistake as this runs non-destructively (at least for now).